### PR TITLE
add initial blog post about GCC

### PR DESCRIPTION
**Pull request by bgruening**
ping @erxleben @stephanflemming

### CODE REVIEW

File: _posts/2019-08-01-gcc.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+![Schedule Thursday](/assets/media/gcc2019/sched_thu.png)
+
+## Freiburg Challenge
+
+![Group Picture](/assets/media/gcc2019/freiburg_challenge.png)
+
+
+## Thank you: Scientific Program Committee
+
+Matthias Bernt, Helmholtz Centre for Environmental Research – UFZ
```
Comment (erasche) [inline _posts/2019-08-01-gcc.md]: Maybe a bulleted list here, they all run together on one line.

File: _posts/2019-08-01-gcc.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+![GCC2019 Logo](/assets/media/gcc2019/gcc_logo_blue_bg.png)
+![Group Picture](/assets/media/gcc2019/grouppicture.jpg)
+
+With 231 participants, GCC2019 was the largest GCC ever with 39 Galaxy training courses, 65 talks, 54 posters,
+13 demos and more than 30 different BoFs. Our participants came from 27 different countries of Eastern- and Western Europe,
+North America, Asia, Oceania and Sub-saharan Africa.
+
+![Group Picture](/assets/media/gcc2019/GCC2019_stats.png)
+
+GCC2019 has started on Monday, 1st July, with a dedicated training day for Galaxy beginners offering a user-centric track with “Introduction to using Galaxy” in parallel to the “Introduction to Galaxy Administration”. Participants interested in long read sequencing and some wet-lab work could visit and try the Nanopore loading zone where the Oxford Nanopore MinION was presented. In the evening, the first Birds-of-a-Feather (BoFs) were offered for special interest groups on Single Cell data analysis, Genome Annotation and Computational Chemistry.
```
Comment (erasche) [inline _posts/2019-08-01-gcc.md]:
```suggestion
GCC2019 has started on Monday, 1st July, with a dedicated training day for Galaxy beginners offering a user-centric track with “Introduction to using Galaxy” in parallel to the admin and developer focused “Introduction to Galaxy Administration”. Participants interested in long read sequencing and some wet-lab work could visit and try the Nanopore loading zone where the Oxford Nanopore MinION was presented. In the evening, the first Birds-of-a-Feather (BoFs) were offered for special interest groups on Single Cell data analysis, Genome Annotation, and Computational Chemistry.
```

File: _posts/2019-08-01-gcc.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+![Schedule Monday](/assets/media/gcc2019/sched_mon.png)
+
+GCC2019 was officially opened on Tuesday, 2nd July, by Björn Grüning and Anika Erxleben. One of our Galaxy power users, Claudia Isabelle Keller Valsecchi, was invited to present her impressive data on “Evolution of X chromosome recognition during Drosophila Dosage Compensation. After some talks on Galaxy Australia, Machine learning, and Galaxy Workflows, talks and workshops were offered in two tracks, one more user-driven, the other more admin-related. Galaxy is used in various fields of research, as we heard e.g. from Anne Fouilloux talking about Climate analysis with Galaxy. During the afternoon break, the participants could discover interesting Galaxy projects in the poster and demo session. The Epigenetics sessions showed how to perform 16S analysis with mothur, ChIPseq data analysis and HiC data analysis. Beside multi-omics data analysis in Galaxy, the first time Machine learning with Galaxy was presented.
```
Comment (erasche) [inline _posts/2019-08-01-gcc.md]:
```suggestion
GCC2019 was officially opened on Tuesday, 2nd July, by Björn Grüning and Anika Erxleben. One of our Galaxy power users, Claudia Isabelle Keller Valsecchi, was invited to present her impressive data on “Evolution of X chromosome recognition during Drosophila Dosage Compensation. After some talks on Galaxy Australia, Machine learning, and Galaxy Workflows, talks and workshops were offered in two tracks, one more user-driven, the other more admin-related. Galaxy is used in various fields of research, as we heard e.g. from Anne Fouilloux talking about Climate analysis with Galaxy. During the afternoon break, the participants could discover interesting Galaxy projects in the poster and demo session. The Epigenetics sessions showed how to perform 16S analysis with mothur, ChIPseq data analysis and HiC data analysis. In addition to multi-omics data analysis in Galaxy, Machine learning with Galaxy was presented for the first time.
```

File: _posts/2019-08-01-gcc.md (hunk 1 above)
Comment (bgruening) [reply _posts/2019-08-01-gcc.md]: Done.

File: _posts/2019-08-01-gcc.md (hunk 2 above)
Comment (bgruening) [reply _posts/2019-08-01-gcc.md]: Sorry, can not take this over, merged it to late :(

**Comment by bgruening** [comment]
@erasche I will keep this open a little bit maybe we get more feedback. Can you redo you changes?

**Comment by erasche** [comment]
Sure

**Comment by erasche** [comment]
Done :)

**Comment by bgruening** [comment]
Merci!

**Comment by bgruening** [comment]
We could add:

* small text about the Freiburg challenge
* GCC program as PDF
* some links to tweets?

We will probably add a new post as soon as the images and videos are ready, but this can be a separate once I think.

File: _posts/2019-08-01-gcc.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+- elixir
+- aws
+---
+
+
+The [2019 Galaxy Community Conference (GCC2019)](https://galaxyproject.org/events/gcc2019) was held in Freiburg, Germany, 1-8 July. 2019 was the largest GCC ever with 231 participants, 39 Galaxy training courses, 65 talks, 54 posters,
+13 demos and more than 30 different BoFs. Our participants came from 27 different countries of Eastern- and Western Europe,
+North America, Asia, Oceania and Sub-saharan Africa.
+
+![GCC2019 Logo](/assets/media/gcc2019/gcc_logo_blue_bg.png)
```
Comment (bgruening) [inline _posts/2019-08-01-gcc.md]: We could also remove the logo, does not fit so nicely here.

**Comment by bgruening** [comment]
@stephanflemming can you please add 1-2 sentence to the Freiburg challenge?

**Comment by erxleben** [comment]
> @stephanflemming can you please add 1-2 sentence to the Freiburg challenge?

done

File: _posts/2019-08-01-gcc.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+- galaxy-europe
+- tiaas
+- denbi
+- elixir
+- aws
+---
+
+
+The [2019 Galaxy Community Conference (GCC2019)](https://galaxyproject.org/events/gcc2019) was held in Freiburg, Germany, 1-8 July. 2019 was the largest GCC ever with 231 participants, 39 Galaxy training courses, 65 talks, 54 posters,
+13 demos and more than 30 different BoFs. Our participants came from 27 different countries of Eastern- and Western Europe,
```
Comment (erasche) [inline _posts/2019-08-01-gcc.md]:
```suggestion
13 demos, and more than 30 different BoFs. Our participants came from 27 different countries of Eastern- and Western Europe,
```

File: _posts/2019-08-01-gcc.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+
+![Stats](/assets/media/gcc2019/GCC2019_stats.png)
+
+Have a look at our [conference book](https://github.com/usegalaxy-eu/gcc2019/raw/master/com/program/book.pdf) to get all information.
+
+GCC2019 started on [Monday](https://gcc2019.sched.com/2019-07-01/overview), July 1st, with a dedicated training day for Galaxy beginners offering a user-centric track with [“Introduction to using Galaxy”](https://gcc2019.sched.com/event/Lud9/introduction-to-using-galaxy-galaxy-101) in parallel to the admin and developer focused [“Introduction to Galaxy Administration”](https://gcc2019.sched.com/event/LudC/introduction-to-galaxy-administration-i). Participants interested in long read sequencing and some wet-lab work could visit and try the [Nanopore loading zone](https://gcc2019.sched.com/event/QZzb/nanopore-loading-zone) with the Oxford Nanopore MinION. In the evening, the first Birds-of-a-Feather (BoFs) were offered for attendees interested in Single Cell data analysis, Genome Annotation, and Computational Chemistry.
```
Comment (erasche) [inline _posts/2019-08-01-gcc.md]:
```suggestion
GCC2019 started on [Monday](https://gcc2019.sched.com/2019-07-01/overview), July 1st, with a dedicated training day for Galaxy beginners offering a user-centric track with [“Introduction to using Galaxy”](https://gcc2019.sched.com/event/Lud9/introduction-to-using-galaxy-galaxy-101) in parallel to the admin and developer focused [“Introduction to Galaxy Administration”](https://gcc2019.sched.com/event/LudC/introduction-to-galaxy-administration-i). Participants interested in long read sequencing and wet-lab work could visit [Nanopore loading zone](https://gcc2019.sched.com/event/QZzb/nanopore-loading-zone) and try the Oxford Nanopore MinION. In the evening, the first Birds-of-a-Feather (BoFs) were offered for attendees interested in Single Cell data analysis, Genome Annotation, and Computational Chemistry.
```

rewrote the bit around visit nanopore, removed 'some' from wet-lab work

File: _posts/2019-08-01-gcc.md [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
...Jonathan](https://gcc2019.sched.com/event/PSfF/galaxy-container-for-hinxton-single-cell-interactive-analysis-portal-hisciap). Running Galaxy for annotation with e.g. [Apollo](https://gcc2019.sched.com/event/PSfO/visual-refinement-of-genome-annotations-with-apollo-in-a-community-environment), [G-ONRamp](https://gcc2019.sched.com/event/PSfR/architectural-and-systems-improvements-to-g-onramp-a-collaborative-eukaryotic-genome-annotation-platform), and  [ChiRA](https://gcc2019.sched.com/event/PSfU/chira-an-integrated-framework-for-chimeric-read-annotation) was successfully shown in talks, posters, trainings and demos. Galaxy also now runs on [Kubernetes](https://gcc2019.sched.com/event/LufN/running-galaxy-on-kubernetes). Before the conference dinner, a public science project was presented by the [Street Science Community](https://gcc2019.sched.com/event/PSfv/street-science-community-make-science-tangible-accessible-by-bringing-it-to-the-street-with-open-science-workshops). More than 150 people joined us for the conference dinner in [Augustiner im Bankepeter](http://www.augustiner-freiburg.de). We spent an amazing evening in the beer garden with our friends and colleagues. Thank you all!
```
Comment (erasche) [inline _posts/2019-08-01-gcc.md]:
```suggestion
[Wednesday\'s](https://gcc2019.sched.com/2019-07-03/overview) Galaxy user track was dedicated to transcriptomics and genomics data analysis. The demand for single cell RNA-seq data analysis was very obvious in the presentations by [Mehmet](https://gcc2019.sched.com/event/PSeu/single-cell-rna-analysis-in-galaxy-on-10x-datasets), [Christophe](https://gcc2019.sched.com/event/PSex/biology-driven-classification-of-single-cells-rnaseq-data-in-galaxy), and [Jonathan](https://gcc2019.sched.com/event/PSfF/galaxy-container-for-hinxton-single-cell-interactive-analysis-portal-hisciap). Running Galaxy for annotation with e.g. [Apollo](https://gcc2019.sched.com/event/PSfO/visual-refinement-of-genome-annotations-with-apollo-in-a-community-environment), [G-ONRamp](https://gcc2019.sched.com/event/PSfR/architectural-and-systems-improvements-to-g-onramp-a-collaborative-eukaryotic-genome-annotation-platform), and [ChiRA](https://gcc2019.sched.com/event/PSfU/chira-an-integrated-framework-for-chimeric-read-annotation) was successfully shown in talks, posters, trainings, and demos. Galaxy also now runs on [Kubernetes](https://gcc2019.sched.com/event/LufN/running-galaxy-on-kubernetes). Before the conference dinner, a public science project was presented by the [Street Science Community](https://gcc2019.sched.com/event/PSfv/street-science-community-make-science-tangible-accessible-by-bringing-it-to-the-street-with-open-science-workshops). More than 150 people joined us for the conference dinner in [Augustiner im Bankepeter](http://www.augustiner-freiburg.de). We spent an amazing evening in the beer garden with our friends and colleagues. Thank you all!
```

oxford comma

**Comment by bebatut** [comment]
Should we mention also that some pictures are there but more will come soon?

**Comment by bgruening** [comment]
> Should we mention also that some pictures are there but more will come soon?

Let's leave this for another blog post? Together with the video recording?

**Comment by bgruening** [comment]
@tnabtaf merge?

**Comment by tnabtaf** [comment]
Yes, but, ...

Give me today to compile some of the feedback.  If I don't post anything before Tuesday morning, then merge it.

**Comment by tnabtaf** [comment]
@bgruening Merge.

**Comment by bgruening** [comment]
We could do a feedback post again or combine it with the picture and/or video post.

**Comment by bgruening** [comment]
Thanks everyone.

**Pull request merged**